typed_python/PySetInstance.cpp

```diff
@@ -601,7 +601,7 @@ void PySetInstance::copyConstructFromPythonInstanceConcrete(SetType* setType, in
             setType->destroy(tgt);
         }
 
-        throw;
+        throw PythonExceptionSet();
```

Comment:
why are we recasting this as a PythonExceptionSet? If the internals threw a std::runtime_error this will not work correctly.

Reply:
My mistake here.